visual-inspection/inspection-frontend/src/components/ExplainInspection.vue

```diff
@@ -27,18 +49,40 @@ export default {
     predictionReady: {
       type: Boolean,
       default: false
+    },
+    isButtonClicked: {
+      type: Boolean,
+      default: false
+    }, 
+    Cls_Acc_List: {
+      type: Object,
+      default: undefined
+    },
+    Cls_Min_Acc :{
+      type: Number,
+      default: 0
     }
+      
   },
   methods: {
-    buttonClicked() {
-      this.$emit('explanation-requested')
+    buttonClicked(index_of_label_to_explain, positive_only_parameter, event) {
+    this.$emit('explanation-requested', index_of_label_to_explain, positive_only_parameter);
+    let elems = document.getElementsByTagName('button');
+    for(let i = 0; i < elems.length; i++)
+      {
+         elems[i].disabled = false;
+      }
+    event.target.disabled = true;
```

Comment:
This should not be necessary.

You can just disable the button(s) directly by using `v-bind:disabled= ...`.